### DISCUSSION TITLE
MinecraftDecompiler v3.3.2 released

### DISCUSSION THREAD

**Discussion by XiaoPangxie732** [MaxPixelStudios/MinecraftDecompiler#125] [General]
A new release of MinecraftDecompiler.

## Dependency Changes
fernflower(233.13135.103 -> 233.14015.106)
asm(9.5 -> 9.6)

## Changelog
- Add a check to ensure the decompiler source type is DIRECTORY when using incremental decompilation (e4464f2)
- Remove snapshot plugin repo, this once used to get the snapshot version of maven-shade-plugin (e4464f2)
- Add an exclusion to asm-tree to force not using it(it has been stripped by maven-shade-plugin previously) (e4464f2)
- Fix tiny v1 bad read (a6b252a)
- Replace `System.getProperty("path.separator")` with `File.pathSeparator` (8b5beaf)
- Directly use constant asm version instead of `XXX.this.api` in inner classes (8b5beaf)
- Nicer exception message for namespaced mappings (59b3397)
- Apply LGPLv3 (#124)

**Milestone: https://github.com/MaxPixelStudios/MinecraftDecompiler/milestone/6**

<hr /><em>This discussion was created from the release <a href='https://github.com/MaxPixelStudios/MinecraftDecompiler/releases/tag/v3.3.2'>MinecraftDecompiler v3.3.2 released</a>.</em>

**Comment by madkarmaa** [comment]
is there any way to import this as a library with Gradle?

**Reply by XiaoPangxie732** [reply]
Absolutely yes. MCD is uploaded to GitHub Packages(https://github.com/MaxPixelStudios/MinecraftDecompiler/packages/597602?version=3.3.2)
You need to follow the instructions to add the dependency, I know it's kinda complex
https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-gradle-registry#using-a-published-package

**Reply by madkarmaa** [reply]
I've tried to understand it but I don't get it

**Reply by madkarmaa** [reply]
nvm, I got it :D
